app/src/main/java/com/zulip/android/networking/AsyncGetEvents.java

```diff
@@ -488,7 +510,10 @@ private void processUpdateMessages(List<EventsBranch> updateEvents) {
         for (EventsBranch event : updateEvents) {
             UpdateMessageWrapper updateEvent = (UpdateMessageWrapper) event;
             Message message = updateEvent.getMessage();
-            if (message != null) {
+
+            // In case of generation of new stream, and the message received from notification bot
```

Comment:
At the time when I work on this (12 days ago) ... I found that the message received from notification bot on stream generation of of type "update_message". This must have been fixed in the server during this time since now I couldn't this event, after reprovisioning by dev env :)